SR2019/2018-10-16-soton-kickstart-confirm.md

```diff
@@ -0,0 +1,15 @@
+---
+to: Student Robotics 2019, Southampton Segment
+subject: Confirming your place at the London Kickstart
+note: This email is to be sent to those who _are_ going to the London kickstart.
+---
+
+Hello!
+
+We'd like to confirm your place at the London kickstart even for Student Robotics 2019.
+
+Exact details and directions are available on our website: https://studentrobotics.org/events/sr2019/london-kickstart/
+
+It is very important that you bring at least 1 laptop for your team. Computers are not available at this location.
```

Comment:
Do we want to suggest we can supply some if they don't have any?

Reply:
No, because as far as I know, we can't. cc @PeterJCLaw

Reply:
I'm not aware that we have a source of laptops we can lend them; certainly I don't think Thread does. I can ask whether we could borrow some, but I'm not even sure we have any spares to consider lending.

If we knew about it in advance and could borrow some from Southampton, that might be an option? Seems like that would need quite a lot of notice and organisation though.

Reply:
It's not often we have issues with this, so given enough notice on it, we should be fine.